engines/bops_api/app/controllers/bops_api/v2/public/documents_controller.rb

```diff
@@ -0,0 +1,29 @@
+# frozen_string_literal: true
+
+module BopsApi
+  module V2
+    module Public
+      class DocumentsController < PublicController
+        def show
+          @planning_application = find_planning_application
+
+          respond_to do |format|
+            format.json
+          end
+        end
+
+        private
+
+        def params
+          # TODO find_planning_application looks for params[:id] but we have params[:planning_application_id]
+          # this is a hack
+          {id: super[:planning_application_id]}
```

Comment:
Could we just define the param as a method

We do in other place

```
  def planning_application_param
    request.path_parameters.key?(:planning_application_id) ? :planning_application_id : :id
  end
  ```
  
  Then can just use the relevant key where needed